tests/files/corosync-authkey

```diff
@@ -0,0 +1,3 @@
+,��`�W���?��h�y+,��-U"�B0����2��!����[>�a!<ɹZ/2���e��Fh�o5П�w�O��iHj�\=��GK	[O�/
+y���7S#r��Zטŷ�:�ٹ���R�L.�" �	��ޯk�o����#]��nҸ��Tݘ
+�O��Q�Iǽo��~
<-��PI9��րA��.w�<�BfK�/�Yc����4*�i��("m���6��u@sQ����	��m;?�
```

Comment:
do you need to add binary files for testing?  It would be better if the test generated these rather than having hardcoded keys checked into git

Reply:
You are absolutely right, fixed in 6ac095a